pages/linux/kde-inhibit.md

```diff
@@ -0,0 +1,15 @@
+# kde-inhibit
+
+> Inhibit various desktop functions while a command runs.
+
+- Inhibit power management:
+
+`kde-inhibit --power {{command}} {{command_arguments}}`
+
+- Inhibit screen saver:
+
+`kde-inhibit --screenSaver {{command}} {{command_arguments}}`
+
+- Inhibit colour correction (night mode) while using VLC:
+
+`kde-inhibit --colorCorrect {{vlc}}`
```

Comment:
Does this save the preference to a file, or run the command itself? Given that this example is slightly different to the previous 2, I'm a little unsure and just want to check :-)

Reply:
No, only inhibit the `night mode` if you start `vlc` using this command, and start the `night mode` when `vlc` exists. It's just a very concrete example.

Reply:
Ah, I see. Thanks!